### DISCUSSION TITLE
DbaTools Certs

### DISCUSSION THREAD

**Discussion by rudyp3** [dataplat/dbatools#9029] [Q&A]
Since the certs are provided by external company – Data Masterminds , do they need to be tracked and updated? Or is it automatically updated as we update the software periodically? How is code signing related to DBATools once installed, I believe it only matters when we are downloading the software to make sure it's legit right? Would we actually have a certificate locally that needs to be updated?

**Comment by wsmelton** [comment]
The certificate works the same way as one used by Pester module or any other code-signing software (nuget package, MSI installer, or PowerShell signed modules). 

There is no action in verifying or installing certificates in your environment by the users. PowerShell does some verification on the certificate chain in the Install-Module process. It offers a way to bypass that by using `-SkipPublisherCheck` if for whatever reason the cert cannot be validated.
